master/getting-started/kubernetes/tutorials/app-layer-policy/index.md

```diff
@@ -0,0 +1,298 @@
+web service does not need, and should not have direct access to the backend database.  The customer 
+web service needs to directly interact with clients outside the cluster, some of whom may be
+malicious.  Unfortunately, vulnerabilities in web applications are all too common.  For example, an
+[unpatched vulnerabiltiy in Apache Struts][struts cve] is what allowed attackers their initial
+access into the Equifax network where they then launched a devastating attack to steal millions of
+people's financial information.
+
+Imagine what would happen if an attacker were to gain control of the customer web pod in our
+application.
+
+Let's simulate this by `exec`'ing into that pod.
+
+    kubectl exec -ti customer-<fill in pod ID> -c customer bash
+
+You should get a bash shell inside the customer pod.  Notice that from here, we get direct access to
+the backend database.  For example, we can list all the entries in the database like this:
+
+    curl http://database:2379/v2/keys?recursive=true | python -m json.tool
+
+(Piping to `python -m json.tool` nicely formats the output.)
+
+#### Single factor authorization
+
+The possession of a key and certificate pair is a very strong assertion that a connection is 
+authentic because it is based on cryptographic proofs that are believed to be nearly impossible to
+forge.  When we authenticate connections this way we can say with extremely high confidence that the
+party on the other end is in possession of the corresponding key. However, this is only a proxy for
+what we actually want to be confident of: that the party on the other end really is the authorized
+workload we want to communicate with.  Keeping the private key a secret is vital to this confidence,
+and occasionally attackers can find ways to trick applications into giving up secrets they should
+not.  For example, the [Heartbleed] vulnerability in OpenSSL allowed attackers to trick an affected
+application into reading out portions of its memory, compromising private keys (among other
+confidential information).
+
+Let's simulate an attacker who has stolen the private keys of another pod.  Since the keys are
+stored as Kubernetes secrets, we won't exploit a vulnerability in a service, but instead just mount
+the secret in a pod that will simulate an attacker.
+
+If you are still `exec`'d into the customer pod, exit out or open a new terminal tab (we will return
+the to the customer pod later). 
+
+```bash
+kubectl apply -f \
+{{site.url}}/{{page.version}}/getting-started/kubernetes/tutorials/app-layer-policy/manifests/20-attack-pod.yaml
+```
+
+Take a look at the [`20-attack-pod.yaml` manifest in your browser](manifests/20-attack-pod.yaml).
+It creates an `ubuntu` pod and mounts `istio.summary` secret.  This will allow us to masquerade as
+if we were the `summary` service, even though this pod is not run as that service account.  Let's
+try this out.  First, `exec` into the pod.
```

Comment:
Here, I think `exec` is OK...it is definitely better than before where you even tried to make it into a gerund. However, there are those who might object to using a command like this as a verb. It's a little dodgy, but I'll leave it to your discretion. You might also want to consider that it is redundant with line 179. Perhaps you could consider again something more in English like "open a bash shell inside"